packages/eslint-config/package.json

```diff
@@ -30,11 +30,13 @@
     "dependencies": {
         "@calm/eslint-plugin-react-intl": "^1.4.1",
         "@comet/eslint-plugin": "workspace:*",
+        "@eslint/eslintrc": "^3.3.1",
```

Comment:
@eslint/eslintrc" dependency missing in eslint-config package.

<img width="1391" height="808" alt="Screenshot 2025-07-10 at 14 22 43" src="https://github.com/user-attachments/assets/34e2e7d5-1c66-4ac1-b4e4-d3293196411d" />

https://github.com/vivid-planet/comet/actions/runs/16194245834/job/45716770099?pr=3352